block/manager.go

```diff
@@ -129,9 +129,9 @@ func (m *Manager) AggregationLoop(ctx context.Context) {
 	height := m.store.Height()
 	var delay time.Duration
 
-	if height == initialHeight {
+	if height < initialHeight {
```

Comment:
@tzdybal I change the condition here to less than.
If there is no block yet, the `height` is 0, `initialHeight` is always >=1, so the condition here should be less than.

Reply:
Related to #699.